src/main.ts

```diff
@@ -0,0 +1,53 @@
+import config from "@config"
+import { Ticker } from '@aliases'
+import tickers from "@tickers"
+import { 
+  fetchTicker, 
+  drawScreen, 
+  getRandomChar, 
+  cursors
+} from '@utils'
+
+const startTime = Date.now()
+
+let count = 0
+async function grab(tickers: Ticker[]) {
+  for (const ticker of tickers) {
```

Comment:
Awesome implementation!

I'm not certain if the aim was to make each API call sequential as to not overload the heroku server but I swapped this logic with a `Promise.all` to essentially get all tickers to update every `REFRESH_RATE_MILLISECONDS` and it seems to be working really well locally.

Something like this:

```js
const fetchData = async (ticker) => {
  try {
    const url = `https://generic709.herokuapp.com/stockc/${ticker}`;
    const response = await fetch(url);
    const data = await response.json();
    return data;
  } catch (err) {
    console.log(error);
  }
};

const grab = async (columnwidth = 20, rows = 30) => {
  const stocks = await Promise.all(
    tickers.map(async (ticker) => {
      return {
        ticker: ticker,
        quote: await fetchData(ticker),
      };
    })
  );

  for (const [idx, { ticker, quote }] of stocks.entries()) {
    if (!quote) {
      return;
    }
    ...

```

Reply:
That's a great idea!

I've updated the code to use ```Promise.all``` and can confirm the performance increase as the requests are now being made concurrently (~850ms vs 15s !)

P.S. we're back to 40 lines of code :)